src/movie/List.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from './Card';
+import { genres, movieSortCritera, movieList } from './Const';
+
+export class MovieList extends Component {
+  constructor(props) {
+    super(props);
+    this.allMovies = movieList;
```

Comment:
why can't you use movieList variable everywhere? why another one(this.allMovies)

Reply:
Actually in future the data will be coming from api and It has to be stored in some variable. Hence i am not using the mock data `movieList` directly.